src/extension.ts

```diff
@@ -198,6 +198,10 @@ class ExtensionManager implements vscode.Disposable {
     return this._folders.get(folder);
   }
 
+  public isActiveFolder(cmt: CMakeToolsFolder): boolean {
+    return this._folders.activeFolder == cmt;
```

Comment:
is double-equals intended here?

Reply:
No, it's a mistake. I'll correct with the next commit.